docs/fast_data/connectors/debezium_cdc.md

```diff
@@ -0,0 +1,420 @@
+---
+id: debezium_cdc
+title: Debezium Server CDC
+sidebar_label: Debezium Server CDC
+---
+
+In this page is describe how to configure Debezium Server to connect with all the supported databases.
+
+## Overview
+
+As previously introduced, in the Mia-Platform Marketplace are available a set of plugins that repackages Debezium Server image, adding the JMX Prometheus Exporter to expose collected metrics regarding the connector.
+
+These plugins are shipped with a predefined configuration tailored for the database of interest to speed up the configuration and deploy phase. This configuration, saved in a file named `application.properties` is mounted as a config map on the path `/debezium/conf` within the container. This leverage the mechanism [Quarkus](https://quarkus.io/) framework, on which Debezium Server is based on, adopts to extend the base configuration of a service.
+
+:::info
+Current Debezium Server plugin requires a dependency on [Redis](https://redis.io/) for storing the metadata regarding the offset and schema management. More details can be found below or in the official Documentation. 
```

Comment:
can we add a link to the official documentation? and also add a link to "can be found below" with the paragraph linked

Reply:
I rephrased this part, since actually there is no clear explanation on the official documentation explaining offsets metadata management.

Reply:
ok, I would put into the same :::note paragraph instead of having the info box and just after the note, since they are related